docs/data-sources/identity_group.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Identity and Access Management (IAM)"
+---
+
+# huaweicloud_identity_group
+
+Use this data source to get the ID of an IAM **User Group**.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_identity_group" "group" {
+  name = "my_group"
+}
+```
+
+## Argument Reference
+
+* `name` - (Optional, String) Specifies the name of the identity group.
+
+* `id` - (Optional, String) Specifies the ID of the identity group.
+
+* `description` - (Optional, String) Specifies the description of the identity group.
+
+## Attributes Reference
+
+* `domain_id` - Indicates the domain the group belongs to.
+
+* `users` - Indicates the users the group contains. Structure is documented below.
+
+The `users` block contains:
+
+* `name` - Indicates the IAM user name.
+
+* `id` - Indicates the ID of the User.
+
+* `description` - Indicates the description of the IAM user.
+
+* `enabled` - Indicates the whether the IAM user is enabled.
+
+* `domain_id` - Indicates the domain the user belongs to.
+
+* `password_expires_at` - Indicates the time when the password will expire.
+  Null indicates that the password has unlimited validity.
+
+* `password_status` - Indicates the password status. True means that the password needs to be changed, and false means that
```

Comment:
The line width exceeds 120, but the lint does not detect it.

Reply:
fixed